.github/workflows/gh-pages.yaml

```diff
@@ -1,9 +1,8 @@
 name: GitHub Pages
 
 on:
-  release:
-    types:
-      - created
+  push:
+    branches: [master, develop]
```

Comment:
**question:** Why updating the documentation based on `develop` branch? That's sound more reliable to stick with the actual event `release`.

Reply:
this change has been reverted